src/cmdlinetest/test_help.t

```diff
@@ -37,6 +37,11 @@ Usage:
                           used multiple times
       -P <property>=<value>
                           Set/ override a property value
+      -x <task>, --exclude=<task>
+                          Exclude optional task dependencies
+      --force-exclude=<task>
+                          Force exclude any task dependencies (dangerous, may
```

Comment:
It's probably better worded as "exclude any task", although "task dependencies" is probably more correct.
But if I run `pyb verify analyze --force-exclude=analyze` then analyze would not run although it's not there by dependency, right?